recipes/vtk-cdat/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "2.10" %}
+{% set name = "vtk-cdat" %}
+
+package:
+    name: {{ name }}
+    version: 7.1.0.{{ version }}
+
+source:
+    git_rev: 1c03d1d03b1c12d99dd090ba4c5ca078e2d9612f
+    git_url: git://github.com/Kitware/VTK.git
+
+build:
+    skip: True  # [win or py3k]
+    number: 0
+    detect_binary_files_with_prefix: true
+    has_prefix_files:  # [win]
+        - Lib/site-packages/vtk/__init__.py  # [win]
```

Comment:
copied from official VTK recipe, hoping to move to `win` eventually, but yes we could remove for now.